### PR TITLE
refactor(page): refactor css

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/1009

### CODE REVIEW

File: src/patternfly/components/Page/page.scss [hunk 1]
```diff
@@ -9,30 +9,50 @@
   // Header brand
   --pf-c-page__header-brand--PaddingTop: var(--pf-global--spacer--sm);
```
Comment (christiemolloy) [inline src/patternfly/components/Page/page.scss]: Looks like you can also remove this as its declared but not used.

File: src/patternfly/components/Page/page.scss [hunk 2]
```diff
@@ -116,8 +161,6 @@
   padding-left: var(--pf-c-page__header-brand--PaddingLeft);
 
   @media (min-width: $pf-global--breakpoint--md) {
-    --pf-c-page__header-brand--PaddingLeft: var(--pf-c-page__header-brand--md--PaddingLeft);
-
     flex: 0 0 var(--pf-c-page__header-brand--md--FlexBasis); // Hardcode header brand width based on design spec, must match --pf-c-nav--Width
     align-self: stretch;
     padding-right: var(--pf-c-page__header-brand--PaddingRight); // set padding right here to allow mobile view to accomodate tools
```
Comment (christiemolloy) [inline src/patternfly/components/Page/page.scss]: should this have md in its name

Comment (mcoker) [reply src/patternfly/components/Page/page.scss]: nice! thanks, updated.

File: src/patternfly/components/Page/page.scss (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/Page/page.scss]: updated

**Pull request merged**